generate/write-to-json-file.js

```diff
@@ -0,0 +1,45 @@
+import path from 'path'
+import fs from 'fs'
+
+const JsonStringifyParameters = {
+  replacer: null,
+  space: 4,
+}
+
+export function writeJsonArray(arrayNewItems, config, applyFilter = false) {
+  const DEST_JSON_FILE = path.join(__dirname, config.destDirectoryFromHere, `${config.destFileName}.json`)
+  const allFileContent = arrayNewItems.map(item => {
+    return applyFilter ? filterProperties(config.propertiesToFilter, item) : item
+  })
+  writeToFile(DEST_JSON_FILE, stringify(allFileContent))
+}
+
+export function writeJsonObject(newItem, config, applyFilter = false) {
+  const DEST_JSON_FILE = path.join(__dirname, config.destDirectoryFromHere, `${config.destFileName}.json`)
+  const itemToCopy = applyFilter ? filterProperties(config.propertiesToFilter, newItem) : newItem
+  writeToFile(DEST_JSON_FILE, stringify(itemToCopy))
+}
+
+function writeToFile(file, content) {
+  fs.writeFile(file, content, err => {
+    if (err) {
+      process.exit(1)
+    }
+  })
+}
+
+function filterProperties(listProps, object) {
+  const filtered = {}
+  listProps.forEach(property => {
+    filtered[property] = object[property]
+  })
+  return filtered
+}
+
+function stringify(thatObject) {
+  const {
+    replacer,
+    space,
+  } = JsonStringifyParameters
+  return JSON.stringify(thatObject, replacer, space)
```

Comment:
Could we just hard code things here? I don't see the reason to have a `JsonStringifyParameters` object. In fact, I don't really see any value in having this function.